components/user-mgt/org.wso2.carbon.user.mgt.ui/src/main/resources/web/role/role-mgt.jsp

```diff
@@ -549,45 +516,41 @@
 
             <%
                 if (roles != null && roles.length > 0 && exceededDomains != null) {
-                    if (exceededDomains.getItemName() != null || exceededDomains.getItemDisplayName() != null) {
+                    if(exceededDomains.getItemName() != null || exceededDomains.getItemDisplayName() != null){
                         String message = null;
-                        if (exceededDomains.getItemName() != null && exceededDomains.getItemName().equals("true")) {
-                            if (exceededDomains.getItemDisplayName() != null &&
-                                !exceededDomains.getItemDisplayName().equals("")) {
+                        if(exceededDomains.getItemName() != null && exceededDomains.getItemName().equals("true")){
```

Comment:
replace '&& exceededDomains.getItemName().equals("true")' to  '&& Boolean.valueOf(exceededDomains.getItemName())'